modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/enums/AndesConfiguration.java

```diff
@@ -33,6 +33,17 @@
 public enum AndesConfiguration implements ConfigurationProperty {
 
     /**
+     * The coordination type. If coordination type is set to "HAZELCAST" cluster communication will be handled
+     * through Hazelcast messages where as if coordination type is set to "RDBMS", it'll be via the database
+     */
+    CLUSTER_EVENT_SYNC_MODE("coordination/clusterCommunicationType", "RDBMS", String.class),
+
+    /**
+     * The interval at which, the cluster events will be read from the database. Specified in seconds.
```

Comment:
Explain what is the effect of having a large/small value for this. on cluster's reliablity/consistency etc. ( that explanation should be in broker.xml as well)